frontend/src/components/pages/topics/create-topic-modal.tsx

```diff
@@ -19,7 +19,7 @@ import {
   CreateTopicRequest_TopicSchema,
   CreateTopicRequest_Topic_ConfigSchema,
 } from 'protogen/redpanda/api/dataplane/v1/topic_pb';
-import { useLegacyCreateTopicMutationWithToast, useLegacyListTopicsQuery } from 'react-query/api/topic';
+import { useCreateTopicMutationWithToast, useListTopicsQuery } from 'react-query/api/topic';
```

Comment:
In here we have an error handling in the modal, why do we want a toast message also, shouldn't we pick one ?

Reply:
I have removed the toasts from mutations across the whole console codebase.